### PR TITLE
Validation with Fiber

### PR DESCRIPTION
**Pull request by asyncfinkd**
Hello, I was looking for validation in your recipes. And I could not find the information I was looking for. That's why I decided to share it with others.

I use [go-playground/validator](https://github.com/go-playground/validator)

### CODE REVIEW

**Review by ReneWerner87 — COMMENTED** [review]
Thx, don't forget a readme file and a link in the readme of the root directory.

**Comment by asyncfinkd** [comment]
> Thx, don't forget a readme file and a link in the readme of the root directory.

Hi Rene, Please check.

File: README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ Here you can find the most **delicious** recipes to cook delicious meals using o
 - [User Auth with JWT](/auth-jwt)
 - [WebSocket Chat Example](/websocket-chat)
 - [WebSockets](/websocket)
+- [Validator](/validator)
```
Comment (ReneWerner87) [inline README.md:51]: Folder name is validation

**Review by ReneWerner87 — CHANGES_REQUESTED** [review]

**Comment by asyncfinkd** [comment]
@ReneWerner87 Fixed :rocket:

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**